doc/builders/packages/etc-files.section.md

```diff
@@ -0,0 +1,18 @@
+# /etc files {#etc}
+
+Certain calls in glibc require access to runtime files found in /etc such as `/etc/protocols` or `/etc/services`
+
+> [getprotobyname](https://linux.die.net/man/3/getprotobyname) is one such function
+
+On non-NixOS distributions these files are typically provided by packages (i.e. _netbase_) if not already pre-installed in your distribution.
+This can cause some non-reproducibility for some code if they rely on these files being present.
+
+The simplest way to add these files to your dependency closure is by exporting the `NIX_ETC_PROTOCOLS` or `NIX_ETC_SERVICES` environment variable pointing it to the [iana-etc](https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.iana-etc.x86_64-linux) package.
```

Comment:
```suggestion
The simplest way to add [iana-etc](https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.iana-etc.x86_64-linux) to your buildInputs:

<example-here>

This will set `NIX_ETC_PROTOCOLS` and `NIX_ETC_SERVICES` environment variables.
```

Reply:
Do you want me to write an example with `buildInputs` And explicitly mention the hook?